velox/type/tests/TimestampConversionTest.cpp

```diff
@@ -104,6 +104,39 @@ TEST(DateTimeUtilTest, fromDateInvalid) {
       1970, 6, 31, "Date out of range: 1970-6-31"));
 }
 
+TEST(DateTimeUtilTest, fromWeekOfMonthDate) {
+  auto daysSinceEpochFromWeekOfMonthDate =
```

Comment:
nit: consider shortening to daysSinceEpoch for readability

Reply:
Thanks. Done